### PR TITLE
Added view helper tests.

### LINKED ISSUE #29: Add tests: view helper functions
Need to make tests for the helper functions located in views.py.

### PR DESCRIPTION
**Pull request by somexpert**
As the name implies, these tests cover the view helper functions that are currently located in views.py. This PR will fix #29.

### CODE REVIEW

**Comment by somexpert** [comment]
ping @ldko, @damonkelley, this is ready for review. Just tests the view helper functions that are found in views.py. The goal will be to later separate these into their own file.

**Comment by somexpert** [comment]
The forms are not tested because there really isn't anything to test that would be outside the scope of testing Django's functionality. And there is also a view which is untested because it is never used (`query_to_tuple`).

**Comment by ldko** [comment]
:+1:

**Comment by damonkelley** [comment]
I am assuming that these are functions that we are going to move to a new module. Can we rename this test module to `test_helpers`? I am guessing we will go for a `helpers.py` instead of `view_helpers.py`

File: tests/test_view_helpers.py [hunk 1]
```diff
@@ -0,0 +1,80 @@
...
+    def test_raises_404(self):
+        with pytest.raises(Http404):
+            views.get_project('fake_project')
+
+
+class TestGetProjectUrlCount():
+
+    def test_returns_surt_count(self):
+        factories.SURTFactory.create_batch(10)
+        # Should not affect the surt count.
```
Comment (damonkelley) [inline tests/test_view_helpers.py]: I don't understand this? Why would it affect the surt count?

Comment (somexpert) [inline tests/test_view_helpers.py]: No, I'm saying that it shouldn't. Basically wherever these comments are, I have created extra objects that should not appear in the results. I do that just to make sure that the we are not getting _more_ than we expect to get in the results.

**Comment by somexpert** [comment]
And yep, you're right, I'll rename this module to `test_helpers`

**Comment by damonkelley** [comment]
:+1:

**Pull request merged**